.kitchen.yml

```diff
@@ -27,6 +27,11 @@ platforms:
   driver:
     box: bento/centos-7.2
 
+  # Windows 2012
```

Comment:
I get a YAML error when trying to `kitchen list` with this, but it goes away if I remove 2 spaces from the beginning of each line. Does that happen for you?